src/pages/StatusPage/index.tsx

```diff
@@ -3,69 +3,86 @@ import { PageStatus, StatusRow } from '../../components';
 import { begin } from '../../services';
 import { Api, ApiWithResponse, Response } from '../../typings/Api';
 import { Container } from './StatusPage.elements';
-
+ 
 interface StatusPageProps {
-  /** An array of API objects */
-  apis: Api[];
-  /** The interval at which to call the APIs in milliseconds */
-  interval?: number;
+ /** An array of API objects */
+ apis: Api[];
+ /** The interval at which to call the APIs in milliseconds */
+ interval?: number;
 }
-
+ 
 interface ApiWithError extends ApiWithResponse {
-  hasError: boolean;
+ hasError: boolean;
 }
-
+ 
 interface StatusPageState {
-  apis: ApiWithError[];
-  hasError: boolean;
+ apis: ApiWithError[];
+ hasError: boolean;
 }
-
+ 
 class StatusPage extends Component<StatusPageProps> {
-  public state: StatusPageState = {
-    apis: [],
-    hasError: false,
-  };
-
-  public handleError = (api: Api, response: Response) => {
-    const apis = [...this.state.apis];
-    const index = apis.findIndex(stateApi => stateApi.api === api);
+ public state: StatusPageState = {
+   apis: [],
+   hasError: false,
+ };
+ 
+ 
+ public handleResponse = (api: Api, response: Response) => {
+   console.log('handleResponse' + response);
+   const apis = [...this.state.apis];
+   const index = apis.findIndex(stateApi => stateApi.api === api);
```

Comment:
Object equality in JS won't work here - has to check a property or it will always be false.

Reply:
I my implementation I did it like this : 
```javascript
const index = apis.findIndex(stateApi => stateApi.api.endpoint === api.endpoint);
```
Should I change here also ?

Reply:
```javascript
const index = apis.findIndex(stateApi => stateApi.api === api);
```
is the way that you do...

Reply:
Yeah, it was probably an oversight on my part - it should probably be updated everywhere. I shall note where to update to make sure we get them all 😄